### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by kgevorkyan**
### What's done:
* Minor fixes: remove temp files, change `projectRootRelativePath` in `saveBinaryFile()` to more correct, logging polishing

### CODE REVIEW

File: save-preprocessor/src/main/kotlin/org/cqfn/save/preprocessor/controllers/DownloadProjectController.kt [hunk 1]
```diff
@@ -229,11 +232,13 @@ class DownloadProjectController(private val configProperties: ConfigProperties,
...
         val pathToProperties = tmpDir.resolve(propertyFile.name)
         propertyFile.copyTo(pathToProperties)
+        propertyFile.delete()
         binFile.copyTo(tmpDir.resolve(binFile.name))
+        binFile.delete()
         val project = executionRequestForStandardSuites.project
         val propertiesRelativePath = pathToProperties.relativeTo(tmpDir).name
         // todo: what's with version?
-        return updateExecution(executionRequestForStandardSuites.project, propertiesRelativePath, binFile.name).flatMap { execution ->
+        return updateExecution(executionRequestForStandardSuites.project, tmpDir.name, binFile.name).flatMap { execution ->
```
Comment (petertrr) [inline save-preprocessor/src/main/kotlin/org/cqfn/save/preprocessor/controllers/DownloadProjectController.kt:241]: It still should be a part of path, not only directory name, Ithink

**Review by petertrr — CHANGES_REQUESTED** [review]

File: save-preprocessor/src/main/kotlin/org/cqfn/save/preprocessor/controllers/DownloadProjectController.kt [hunk 2]
```diff
@@ -126,6 +127,7 @@ class DownloadProjectController(private val configProperties: ConfigProperties,
     /**
      * Controller to download standard test suites
      */
+    @ExperimentalFileSystem
```
[the remark this answers is not in the document]
Comment (kgevorkyan) [reply save-preprocessor/src/main/kotlin/org/cqfn/save/preprocessor/controllers/DownloadProjectController.kt]: Fixed

File: save-preprocessor/src/main/kotlin/org/cqfn/save/preprocessor/controllers/DownloadProjectController.kt [hunk 3]
```diff
@@ -134,13 +136,14 @@ class DownloadProjectController(private val configProperties: ConfigProperties,
             val tmpDir = generateDirectory(testSuiteUrl)
             cloneFromGit(GitDto(testSuiteUrl), tmpDir)?.use {
                 Flux.fromIterable(testSuitePaths).flatMap { testRootPath ->
-                    log.info("Starting to discover root test config for test root $testRootPath")
+                    log.info("Starting to discover root test config in test root path: $testRootPath")
                     val testResourcesRootAbsolutePath = tmpDir.resolve(testRootPath).absolutePath
                     val rootTestConfig = testDiscoveringService.getRootTestConfig(testResourcesRootAbsolutePath)
-                    log.info("Starting to discover standard test suites for config test root $testRootPath in $testResourcesRootAbsolutePath")
-                    val tests = testDiscoveringService.getAllTestSuites(null, rootTestConfig, "stub")
+                    log.info("Starting to discover standard test suites for root test config ${rootTestConfig.location.name} in $testResourcesRootAbsolutePath")
```
[the remark this answers is not in the document]
Comment (kgevorkyan) [reply save-preprocessor/src/main/kotlin/org/cqfn/save/preprocessor/controllers/DownloadProjectController.kt]: This message already contain `testResourcesRootAbsolutePath`, however I changed this

File: save-preprocessor/src/main/kotlin/org/cqfn/save/preprocessor/controllers/DownloadProjectController.kt (hunk 1 above)
Comment (kgevorkyan) [reply save-preprocessor/src/main/kotlin/org/cqfn/save/preprocessor/controllers/DownloadProjectController.kt:241]: This path used like `execution.resourcesRootPath` in `buildBaseImageForExecution()` in this way:
```kotlin
        val resourcesPath = File(
            configProperties.testResources.basePath,
            execution.resourcesRootPath,
        )
```

So we take the `configProperties.testResources.basePath` and then appends the relative path, which is name of directory, so I think all is correct.
Or what did you mean?

Comment (petertrr) [reply save-preprocessor/src/main/kotlin/org/cqfn/save/preprocessor/controllers/DownloadProjectController.kt:241]: I actually meant, that there can be several directories, and `name` returns the name of the last one only. But as I see now, in this function we don't really pay attention to test root directory and just copy everything into root. Then, `tmpDir.name` works, but we will need to change general logic later.

**Review by petertrr — APPROVED** [review]

**Pull request merged**